x/staking/types/params.go

```diff
@@ -33,12 +33,27 @@ const (
 )
 
 var (
-	KeyUnbondingTime     = []byte("UnbondingTime")
-	KeyMaxValidators     = []byte("MaxValidators")
-	KeyMaxEntries        = []byte("MaxEntries")
-	KeyBondDenom         = []byte("BondDenom")
-	KeyHistoricalEntries = []byte("HistoricalEntries")
-	KeyPowerReduction    = []byte("PowerReduction")
+	// ValidatorBondFactor of -1 indicates that it's disabled
+	ValidatorBondDisabled = sdk.NewDecFromInt(sdk.NewInt(-1))
+
+	// DefaultValidatorBondFactor is set to -1 (disabled)
+	DefaultValidatorBondFactor = ValidatorBondDisabled
+	// DefaultGlobalLiquidStakingCap is set to 100%
```

Comment:
Do you mean the decision to leave them disabled by default?

Reply:
DefaultGlobalLiquidStakingCap = 100% means that all the staked assets can be liquid, right?

Reply:
I don't think it was that intentional - but at the time, the thinking was probably that it didn't make sense to use gaia's specific params here as defaults (since this was originally written to be used with other chains)

Reply:
Setting DefaultGlobalLiquidStakingCap to 100% was intentional. The Cosmos Hub does not currently enforce a limit on the maximum share of voting power that can be held by liquid staking providers, so the de facto cap is currently 100%. That's why we left the parameter as it is: a chain can adopt the LSM, and decide a cap in their upgrade handler through governance. The Cosmos hub voted in the LSM proposal to set the parameter to 25%. That's reflected in the Hub v11 upgrade handler. I don't think the Cosmos Hub voting for that parameter value should make it the default for all chains.